AdminServer/appscale/admin/operation.py

```diff
@@ -8,8 +8,9 @@
 from .constants import Types
 
 
-class CreateVersionOperation(object):
+class Operation(object):
   """ A container that keeps track of deployment operations. """
```

Comment:
Should the docstring be changed to reflect that this is a parent class/interface?